[2023-06-29T20:20:27.992+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data.transfer_s3_to_rds manual__2023-06-29T16:19:51-04:00 [queued]>
[2023-06-29T20:20:28.002+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data.transfer_s3_to_rds manual__2023-06-29T16:19:51-04:00 [queued]>
[2023-06-29T20:20:28.002+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-29T20:20:28.002+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-29T20:20:28.002+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-29T20:20:28.037+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): transfer_s3_to_rds> on 2023-06-29 20:19:51+00:00
[2023-06-29T20:20:28.044+0000] {standard_task_runner.py:55} INFO - Started process 2955 to run task
[2023-06-29T20:20:28.047+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data', 'transfer_s3_to_rds', 'manual__2023-06-29T16:19:51-04:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data.py', '--cfg-path', '/tmp/tmpj2xqe7_k']
[2023-06-29T20:20:28.049+0000] {standard_task_runner.py:83} INFO - Job 39: Subtask transfer_s3_to_rds
[2023-06-29T20:20:28.147+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data.transfer_s3_to_rds manual__2023-06-29T16:19:51-04:00 [running]> on host b4b1b6c8b5f0
[2023-06-29T20:20:28.508+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data
AIRFLOW_CTX_TASK_ID=transfer_s3_to_rds
AIRFLOW_CTX_EXECUTION_DATE=2023-06-29T20:19:51+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-29T16:19:51-04:00
[2023-06-29T20:20:28.524+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-29T20:20:28.525+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-29T20:20:31.216+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-29T20:20:50.002+0000] {logging_mixin.py:137} INFO - End to add rows! 10000 rows added.
[2023-06-29T20:20:50.006+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-29T20:20:50.044+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data, task_id=transfer_s3_to_rds, execution_date=20230629T201951, start_date=20230629T202027, end_date=20230629T202050
[2023-06-29T20:20:50.178+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-29T20:20:50.261+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
